content-webapp/src/main/webapp/vue-app/news-details/components/ExoNewsDetails.vue

```diff
@@ -73,9 +73,15 @@
         :cancel-label="$t('news.button.cancel')"
         @ok="deleteNews" />
       <exo-news-edit-publishing-drawer
-        v-if="news && currentUser"
+        v-if="news && currentUser && !newPublicationDrawerEnabled"
         :news="news"
         @refresh-news="getNewsById(newsId)" />
+      <note-publication-drawer
```

Comment:
this component should be loaded only when FF is active

Reply:
Done